### PR TITLE
Backup and DB variables automated

### PR DESCRIPTION
**Pull request by plavy**

### CODE REVIEW

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml [hunk 1]
```diff
@@ -8,41 +8,18 @@ global:
   env:
     PROJECT_SLUG: '{{cookiecutter.project_slug}}'
 {% raw %}
-    ENVIRONMENT_NAME: {{ env.ENVIRONMENT_NAME }}
     ENVIRONMENT: '{{ env.ENVIRONMENT | default("local", True) }}'
```
Comment (plavy) [inline project_template/{{cookiecutter.project_repo_name}}/config/config.yaml:11]: Do we need this variable? Is it local by default?

Comment (plavy) [inline project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: @lbenc135  Do we need this variable? Is it local by default?

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml [hunk 2]
```diff
@@ -8,41 +8,18 @@ global:
...
-    BACKUPS_STORAGE_ACCOUNT_NAME: '{{ env.BACKUPS_STORAGE_ACCOUNT_NAME }}'
-
     # Misc
     DEFAULT_TIME_ZONE: Europe/Zagreb
 
   secrets:
     SENTRY_DSN: '{{ env.SENTRY_DSN }}'
 
-    MONGO_CLIENT_CONNECTION_STRING: '{{ env.MONGO_CLIENT_CONNECTION_STRING }}'
     MONGO_CLIENT_PASSWORD: admin
```
Comment (plavy) [inline project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: @lbenc135 These should be set in CI/CD variables, do they maybe have some defaults for local run?

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml (hunk 1 above)
Comment (lbenc135) [reply project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: we don't, yes it's local by default

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml [hunk 3]
```diff
@@ -8,41 +8,16 @@ global:
...
-    REDIS_CLIENT_PORT: '6379'
-
-    BACKUP_CONTAINER_NAME: '{{ env.BACKUP_CONTAINER_NAME }}'
-    BACKUPS_STORAGE_ACCOUNT_NAME: '{{ env.BACKUPS_STORAGE_ACCOUNT_NAME }}'
-
     # Misc
     DEFAULT_TIME_ZONE: Europe/Zagreb
 
   secrets:
-    SENTRY_DSN: '{{ env.SENTRY_DSN }}'
```
Comment (lbenc135) [inline project_template/{{cookiecutter.project_repo_name}}/config/config.yaml:37]: do you set this automatically as well?

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml (hunk 2 above)
Comment (lbenc135) [reply project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: yes, they're all empty by default, which works locally

File: project_template/{{cookiecutter.project_repo_name}}/config/local.config.yaml [hunk 4]
```diff
@@ -1,25 +1,10 @@
...
-    REDIS_CLIENT_HOST: {{ env.REDIS_CLIENT_HOST }}
-    REDIS_CLIENT_PORT: {{ env.REDIS_CLIENT_PORT }}
-
     INTERNAL_CONTROL_PANEL_URL: '{{ env.CONTROL_PANEL_URL | default("http://localhost:8100", true) }}'
 
   secrets:
-    MONGO_CLIENT_CONNECTION_STRING: {{ env.MONGO_CLIENT_CONNECTION_STRING }}
     MONGO_CLIENT_PASSWORD: {{ env.MONGO_CLIENT_PASSWORD }}
-
     POSTGRES_CLIENT_PASSWORD: {{ env.POSTGRES_CLIENT_PASSWORD }}
```
Comment (lbenc135) [inline project_template/{{cookiecutter.project_repo_name}}/config/local.config.yaml]: you can remove them here as well

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml (hunk 3 above)
Comment (plavy) [reply project_template/{{cookiecutter.project_repo_name}}/config/config.yaml:37]: Yes, `MONGO_CLIENT_CONNECTION_STRING`, `MONGO_CLIENT_PASSWORD`, `POSTGRES_CLIENT_PASSWORD`, `SENTRY_DSN` are loaded into global-secret if they exist.

File: project_template/{{cookiecutter.project_repo_name}}/config/config.yaml (hunk 2 above)
Comment (lbenc135) [reply project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: @plavy how do we keep this backwards compatible? does the new CI auto-generate random password?

Comment (plavy) [reply project_template/{{cookiecutter.project_repo_name}}/config/config.yaml]: You can always override CI/CD variables with config

**Pull request merged**